### PR TITLE
Full Ex2 Attached - Remember to Adj. the File Path!

### PR DESCRIPTION
**Pull request by ps2016-mt**

### CODE REVIEW

File: load_clean.py [hunk 1]
```diff
@@ -20,7 +23,9 @@ def load_csv(file_path):
     return df
 
 # Call the function and store the result in the gdp_data variable
-gdp_data = load_csv("C:\\Users\\psingh\\Desktop\\Personal\\Cambridge\\Year 1\\D100\\gdp_rep0_structure\\gdp_data.csv")
+gdp_data = load_csv(r"C:/Users\psingh\Desktop\gdp_repo_structure\gdp_data.csv")
```
Comment (yuanjingshi) [inline load_clean.py:26]: The path is specific to your local machine with your username on it. It is not portable on other machines.

File: load_clean.py [hunk 2]
```diff
@@ -38,20 +43,39 @@ def clean_data(df):
...
-    
-    # Select only the relevant year columns'
+
+    # Select only the relevant year columns and 'Country' column
     country_column = 'Country'
-    
     columns_to_keep = [country_column] + year_columns
     df_clean = df[columns_to_keep]
 
     print(df_clean.head(8))
```
Comment (yuanjingshi) [inline load_clean.py:54]: Consider replace the print function with log function to help debug. Generally it is not a good practice to leave print in the function.

File: load_clean.py (hunk 1 above)
Comment (ps2016-mt) [reply load_clean.py:26]: Apologies, will fix that, thanks!

File: load_clean.py (hunk 2 above)
Comment (ps2016-mt) [reply load_clean.py:54]: Thanks for the tip, will look to implement

**Pull request merged**